packages/odie-client/src/context/index.tsx

```diff
@@ -96,7 +96,9 @@ export const OdieAssistantProvider: React.FC< OdieAssistantProviderProps > = ( {
 	 * The main chat thread.
 	 * This is where we manage the state of the chat.
 	 */
-	const { mainChatState, setMainChatState } = useGetCombinedChat( canConnectToZendesk );
+	const { mainChatState, setMainChatState } = useGetCombinedChat(
+		isUserEligibleForPaidSupport && canConnectToZendesk
```

Comment:
This will probably break the work that you added here https://github.com/Automattic/wp-calypso/pull/99604 because those users are not eligible.

Reply:
The eligibility is passed down from [here](https://github.com/Automattic/wp-calypso/blob/0d73cb98968bca2e3f20da553efacc3653a72779/packages/help-center/src/components/help-center-content.tsx#L71)
I didn't test this case but should work

Reply:
Ah brilliant!